inc/commonitiltask.class.php

```diff
@@ -1248,16 +1249,50 @@ static function genericDisplayPlanningItem($itemtype, array $val, $who, $type =
          }
       }
 
-      if (isset($val["state"])) {
-         $html.= "<span>";
-         $html.= Planning::getState($val["state"]);
-         $html.= "</span>";
-      }
-      $html.= "<div>";
-      $html.= sprintf(__('%1$s: %2$s'), __('Priority'), $parent->getPriorityName($val["priority"]));
-      $html.= "</div>";
-      $html.= "<div class='event-description rich_text_container'>".html_entity_decode($val["content"])."</div>";
-      $html.= $recall;
+      switch ($options['view_name']) {
```

Comment:
It seems this part of code is copied multiple time in different classes.
It should be better adding a function to format the part (in `PlanningEvent` trait for example) and calling it in `*DisplayPlanningItem*` functions.

This to avoid future fixes/evolutions in multiple places.

Reply:
corrected